src/main/java/frc/robot/BuildConstants.java

```diff
@@ -8,11 +8,11 @@ public final class BuildConstants {
   public static final String MAVEN_NAME = "template-talon_standard";
```

Comment:
In general, we shouldn't have auto-generated files in git

Reply:
Seems to have been resolved.